winbuild/build_prepare.py

```diff
@@ -257,10 +259,10 @@ def cmd_msbuild(
         "libs": [r"bin\*.lib"],
     },
     "libimagequant": {
-        # commit: Merge branch 'master' into msvc (matches 2.16.0 tag)
-        "url": "https://github.com/ImageOptim/libimagequant/archive/f41ee301ff3a407b16991af3dbe03910919bbdc3.zip",  # noqa: E501
-        "filename": "libimagequant-f41ee301ff3a407b16991af3dbe03910919bbdc3.zip",
-        "dir": "libimagequant-f41ee301ff3a407b16991af3dbe03910919bbdc3",
```

Comment:
Why switch to the master branch here? As the comment says, previously this used the msvc branch, so the 2.17.0 equivalent would be https://github.com/ImageOptim/libimagequant/commit/e4c1334be0eff290af5e2b4155057c2953a313ab

Reply:
I've updated to the 2.17.0 version of the msvc branch in #5916